modules/finality-verifier/src/lib.rs

```diff
@@ -70,7 +70,7 @@ pub mod pallet {
 	use frame_system::pallet_prelude::*;
 
 	#[pallet::config]
-	pub trait Config: frame_system::Config {
+	pub trait Config<I: 'static = ()>: frame_system::Config {
```

Comment:
is `I: 'static = ()` equal `I: DefaultInstance`?

Reply:
I guess so :) From what I've learned from this new pallet declaration macros docs, now 'instance' should only be `'static`. Previously, with `decl_module! {}`, it had to implement `Instance` trait. I've been copypasting this from the `balances` pallet, which atm is the only pallet that uses this new pallet declaration syntax and supports multiple instances. There, `()` is used. Tbh, I'm not even sure if `DefaultInstance` is still being generated by the `#[pallet]` :)

Reply:
cool, thanks a lot!

Reply:
So according to Gui `DefaultInstance` is not being supported anymore, and `()` should be used as the "default".